### DISCUSSION TITLE
Define a default project location

### DISCUSSION THREAD

**Discussion by kevinruscoe** [godotengine/godot-proposals#5145] [Editor]
On OS X, when you create a new project, its default location is the users home folder (~). I always have to change this to ~/Code/Games/Godot and create the new game folder inside there. It would be good if we could set a default which is used.

**Comment by Calinou** [comment]
This is already possible. You can change Default Project Path in the editor settings after creating a project.

As for remembering the last location used to create a project, see https://github.com/godotengine/godot-proposals/issues/1312.

**Reply by kevinruscoe** [reply]
Thanks for this! I wasn't expecting it to be in the editor settings, I was expecting it to be in the "new project" window.
